open-apparel-registry/open-apparel-registry
# Correct contributor types

## Overview

A recent update to the contributor types introduced a misspelling
of 'Organization' as 'Organisation'. This corrects the typo.

## Testing Instructions

* Run `./scripts/manage migrate` and confirm the migration runs as expected
* Visit http://localhost:8081/admin/api/contributor/8/change/ and confirm the `contrib_type` is correctly displayed. 

## Checklist
